pkg/reconciler/ingress/ingress.go

```diff
@@ -76,8 +77,10 @@ type Reconciler struct {
 	virtualServiceLister istiolisters.VirtualServiceLister
 	gatewayLister        istiolisters.GatewayLister
 	secretLister         corev1listers.SecretLister
+	svcLister            corev1listers.ServiceLister
 
 	tracker   tracker.Interface
+	gwTracker tracker.Interface
```

Comment:
It is difficult for me to understand what this value is for :thinking:  Is it necessary?

Reply:
good catch. it is not necessary. deleted it.